homeassistant/helpers/service.py

```diff
@@ -112,3 +116,46 @@ def extract_entity_ids(hass, service_call, expand_group=True):
             return [service_ent_id]
 
         return service_ent_id
+
+
+@bind_hass
+def get_description(hass, domain, service):
+    """Return the description (i.e. user documentation) for a service call."""
+    if domain == ha.DOMAIN:
+        import homeassistant.components as components
+        comp_file = components.__file__
+    else:
+        comp_file = get_component(domain).__file__
+
+    descriptions = DESCRIPTION_CACHE.get(comp_file)
+    if descriptions is None:
+        descriptions = load_yaml_config_file(
+            path.join(path.dirname(comp_file), 'services.yaml'))
+        DESCRIPTION_CACHE[comp_file] = descriptions
```

Comment:
Would it make sense to keep a cache per file or would a cache per service be better?